Docker/dev.pgsql.Dockerfile

```diff
@@ -11,14 +11,14 @@ FROM mcr.microsoft.com/dotnet/sdk:8.0.403-alpine3.20@sha256:07cb8622ca6c4d7600b4
 ARG ASPNETCORE_ENVIRONMENT=${ASPNETCORE_ENVIRONMENT:-"Production"}
 
 WORKDIR /source
-COPY Application/NuGet.Config EdFi.Ods.AdminApi/
-COPY Application/EdFi.Ods.AdminApi EdFi.Ods.AdminApi/
+COPY --from=assets ./Application/NuGet.Config EdFi.Ods.AdminApi/
```

Comment:
There is a Check Failure showing below: `DL3022 warning: `COPY --from` should reference a previously defined `FROM` alias`. This is a false positive. Apparently the linter does not know about using `--from` with the value provided at the command line using `--build-context`.